content/rule_book/index.md

```diff
@@ -1,9 +1,12 @@
-# Rule Book
+# Bible Quizzing Rule Book
+
+This is the official rule book for Bible Quizzing, which represents what will be followed at the annual International Bible Quizzing championship meet. It is also a recommendational document for Quizzing districts and any other organizations interested in Bible Quizzing. These districts and other organizations retain the right to set and enforce their own rules using whatever mechanisms they deep most appropriate.
```

Comment:
"recommendational" is not a word?
"deep" -> "deem"

Reply:
Deem derp. :) I'm going to claim "recommendational" is a word, though.